src/components/structures/CustomRoomTagPanel.js

```diff
@@ -30,7 +30,7 @@ class CustomRoomTagPanel extends React.Component {
         };
     }
 
-    componentWillMount() {
+    componentDidMount() {
         this._tagStoreToken = CustomRoomTagStore.addListener(() => {
```

Comment:
in the case of a listener like this, what if an event fires between construction (will-mount) and did-mount, it'll be missed entirely, no?

Reply:
oh probably, phase 2 of this PR series is to go through and fix components to be classes. This particular one is already a class, but it would also get fixed in the mass conversion.

Reply:
Alright, as long as we track looking through every single componentDidMount and moving things to ctor if it makes sense to.

Reply:
yup, that's the plan. 

It's a world of sadness I've invented for myself.

Reply:
:((